### PR TITLE
Add manual section for Foreman 3.9

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
Adding the manual section for Foreman 3.9 in preparation for branching.

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
I still had some notes to add precise instructions to the procedure, so I ended up finishing those: https://github.com/theforeman/theforeman-rel-eng/pull/307

**Review by ekohl — APPROVED** [review]
Leaving it open until we actually do the branching, but the content looks good.

File: _includes/manuals/3.9/4.3.4_smartproxy_dhcp.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+:dhcp_leases: /var/lib/dhcp3/dhcpd.leases
+```
+
+<div class="alert alert-info">The foreman-proxy account must be able to read both configuration files. In particular, check the permissions on the parent directory (e.g. /etc/dhcp) permit world read/execute.</div>
+
+If the DHCP server is secured with an "omapi_key", the following entries must be set with the same values:
+
+```yaml
+:key_name: omapi_key
+:key_secret: XXXXXXXX
```
Comment (kenyon) [inline _includes/manuals/3.9/4.3.4_smartproxy_dhcp.md:66]:
````suggestion
:key_secret: XXXXXXXX
```
````

Fixed this for the other versions in #2118.

**Review by kenyon — CHANGES_REQUESTED** [review]

File: _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+* enable dhcp libvirt provider
+* default settings for both providers are sufficient (network named `default`
+  on local libvirt daemon instance)
+
+Important configuration values are, in `tftp.yml`:
+
+```yaml
+:tftp: true
+:tftproot: /var/tftproot
+:tftp_servername: 192.168.122.1
```
Comment (kenyon) [inline _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md:78]:
````suggestion
:tftp_servername: 192.168.122.1
```
````

File: _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+in `dns.yml`:
+
+```yaml
+:enabled: true
+:use_provider: dns_libvirt
```
Comment (kenyon) [inline _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md:85]:
````suggestion
:use_provider: dns_libvirt
```
````

File: _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+in `dhcp.yml`:
+
+```yaml
+:enabled: true
+:use_provider: dhcp_libvirt
```
Comment (kenyon) [inline _includes/manuals/3.9/4.3.11_smartproxy_libvirt.md:92]:
````suggestion
:use_provider: dhcp_libvirt
```
````

**Review by kenyon — CHANGES_REQUESTED** [review]
More fixes from #2119.

**Comment by kenyon** [comment]
The changes from #2120, #2121, and #2122 should be added too.

File: _includes/manuals/3.9/1.2_release_notes.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+#### Smart Proxy - Tests
+* Allow clearing the log buffer ([#34163](https://projects.theforeman.org/issues/34163))
+
+*A full list of changes in 3.9.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12]=1765)*
+
+### Contributors
+
+We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
+
+Adam Růžička,Alexey Masolov,Archana Kumari,Areyus,Bastian Schmidt,Dyrkon,Eric Helms,Evgeni Golov,Ewoud Kohl van Wijngaarden,Girija Soni,Griffin Sullivan,Ian Ballou,Jeremy Lenz,Joshua Hoblitt,Karolina Malyjurkova,Laura Hild,Leos Stejskal,Manisha Singhal,Marek Hulán,Maria Agaphontzev,Markus Bucher,Maximilian Kolb,Nikos Moumoulidis,Nofar Alfassi,Oleh Fedorenko,Partha Aji,Pat Riehecky,Patrick Creech,Romain Tartière,Ron Lavi,Samir Jha,Sebastian Bublitz,Shimon Shtein,Tim Meusel,Tony James,Trent Anderson,William Bradford Clark
```
Comment (ekohl) [inline _includes/manuals/3.9/1.2_release_notes.md]: Somehow the spaces were lost. I think that was in my rewrite.

File: _includes/manuals/3.9/3.3.1_rpm_packages.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+### Available packages
+
+Install foreman and other foreman-* packages to add functionality:
+
+    foreman               Foreman server
+    foreman-proxy         Foreman Smart Proxy
+    foreman-debug         Log and debug collection tools
+    foreman-ec2           Amazon EC2 provisioning support
+    foreman-gce           Google Compute Engine provisioning support
```
Comment (ekohl) [inline _includes/manuals/3.9/3.3.1_rpm_packages.md:44]: We dropped this package some time ago and should drop it.

File: _includes/manuals/3.9/3.3.2_debian_packages.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+* foreman-postgresql
+
+Optional functionality:
+
+* foreman-console
+* foreman-debug
+* foreman-dynflow-sidekiq
+* foreman-ec2
+* foreman-gce
```
Comment (ekohl) [inline _includes/manuals/3.9/3.3.2_debian_packages.md:64]: Same here

**Review by ekohl — COMMENTED** [review]
It may be better to copy everything fresh again after https://github.com/theforeman/theforeman.org/pull/2123 is merged.

File: _includes/manuals/3.9/1.2_release_notes.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+* Add Dependabot Configuration ([#36899](https://projects.theforeman.org/issues/36899))
+* Run foreman-proxy.service with PrivateTmp ([#36890](https://projects.theforeman.org/issues/36890))
+* Raise minimum Ruby version to 2.7 ([#36779](https://projects.theforeman.org/issues/36779))
+
+#### Smart Proxy - Tests
+* Allow clearing the log buffer ([#34163](https://projects.theforeman.org/issues/34163))
+
+*A full list of changes in 3.9.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12]=1765)*
+
+### Release notes for 3.9.1
```
Comment (ekohl) [inline _includes/manuals/3.9/1.2_release_notes.md]: We list it from newest to oldest, so first 3.9.1 and then 3.9.0

**Review by ekohl — CHANGES_REQUESTED** [review]

File: _includes/manuals/3.9/1.2_release_notes.md (hunk 8 above)
Comment (Griffin-Sullivan) [reply _includes/manuals/3.9/1.2_release_notes.md]: ok switched it back

**Review by ekohl — APPROVED** [review]
Feel free to merge. Note that this will show a banner when viewing 3.7 docs that it's EOL and 3.9 is the latest. However, 3.9 won't show up until you update this line: https://github.com/theforeman/foreman-infra/blob/5f51dcae45dc21af1aa463249c8aa40da0a56cfa/puppet/modules/profiles/manifests/web.pp#L22

Instead, it will redirect to nightly.

**Comment by Griffin-Sullivan** [comment]
Ok I'll update the infra. BTW I added one change after your approval to update `foreman_latest_releases`

**Comment by Griffin-Sullivan** [comment]
@ekohl here's the PR https://github.com/theforeman/foreman-infra/pull/2010

**Review by ekohl — APPROVED** [review]

**Pull request merged**